projects/msvc16/unittest/stdafx.h

```diff
@@ -0,0 +1,13 @@
+// stdafx.h : include file for standard system include files,
+// or project specific include files that are used frequently, but
+// are changed infrequently
+//
+
+#pragma once
+
+#include "targetver.h"
+
+// Headers for CppUnitTest
+#include "CppUnitTest.h"
+
+// TODO: reference additional headers your program requires here
```

Comment:
__[Goodcheck]__ Complete all work!
<details>


Based on the comments it looks like some work was left for later.
Are you sure you want to continue?

</details>
<sub>posted by Sider</sub>